game/tests/test_random_road.py

```diff
@@ -121,9 +121,10 @@ def test_branchiness_min(self):
             self.assertTrue(len(node['connectedNodes']) <= 2)
 
     def test_branchiness_max(self):
-        """ Test that if the branchiness > 0 we get branches. """
+        """ Test that if the branchiness = 1 we get branches. """
 
-        data = self.create_test_data(branchiness=2, loopiness=0)
+        number_of_tiles = (random.randint(10, 40))
```

Comment:
This test has too many random variants for a single path.
1. The test needs to be repeatable. If it fails it should fail on every run, or we should at least be able to see the generated case where it failed, and be able to repeat it.
2. If the test cannot be repeatable. Then make sure to exercise the randomness to get a significant result, meaning rerun the random generator to cover different outputs.
3. Fix the number of tiles, or run a few sizes (and repeat the random level generator).
4. Optionally, Fail the test on the first failure, instead of repeating the loop after a fail.

This applies to all the tests that use randomly generated levels.